packages/shared-state-bat_links_info/files/etc/uci-defaults/shared-state_bat_links_info_cron

```diff
@@ -4,10 +4,17 @@ unique_append()
 {
 	grep -qF "$1" "$2" || echo "$1" >> "$2"
 }
-
+## information generator datatype
 uci set shared-state.bat_links_info=dataType
 uci set shared-state.bat_links_info.name='bat_links_info'
 uci set shared-state.bat_links_info.scope='community'
 uci set shared-state.bat_links_info.ttl='2400'
 uci set shared-state.bat_links_info.update_interval='30'
 uci commit shared-state
```

Comment:
This line needs to be moved after the last `uci set` at line 20 otherwise reference state configurations are not guaranteed to be persisted across reboot